### PR TITLE
Update uxmhf x86 docs

### PR DESCRIPTION
**Pull request by slab14**
Updates to x86 docs, noting some changes (adding additional clarifying details) based upon comments in the uberspark.org forums.

### CODE REVIEW

File: docs/pc-intel-x86_32/supported-os.rst [hunk 1]
```diff
@@ -13,5 +13,10 @@ long as it:
 The following guest OSes are known to work:
 
 
-* Ubuntu 16.04 LTS with Linux Kernel 4.4.x (CONFIG_X86_PAE=n) and below
+* Ubuntu 16.04 LTS with Linux Kernel 4.4.x (#CONFIG_X86_PAE is not set) and below
 * Ubuntu 12.04 LTS with Linux Kernel 3.2.0-27-generic and below
+
+Note, that currently certain kernel modules must be black-listed (e.g., in `//etc/modprobe.d/blacklist.conf`):
```
Comment (amitvasudevan) [inline docs/pc-intel-x86_32/supported-os.rst]: there is an extra `/` within `//etc...`?

**Review by amitvasudevan — COMMENTED** [review]
Can you also please add documentation to describe the new configure switch `--with-debug-serial-maxcpus` ? This should be equal to the number of cores on your platform when debugging is enabled. You can add this to the "verifying and building" section of pc-intel-x86_32

**Comment by amitvasudevan** [comment]
also would be nice to have a full grub menu.lst listing: a small concise self-contained listing example which shows all the different elements; default OS, uberXMHF, savedefault etc.

File: docs/pc-intel-x86_32/supported-os.rst (hunk 1 above)
Comment (slab14) [reply docs/pc-intel-x86_32/supported-os.rst]: thanks, removed

**Pull request merged**